test/test_nn.py

```diff
@@ -2897,6 +2901,26 @@ def test_tdparams_clone(self):
             assert (val == td.get(key)).all()
 
 
+class TestAddStateIndependentNormalScale:
+    def test_add_scale_basic(self, num_outputs=4):
+        module = nn.Linear(3, num_outputs)
```

Comment:
Should this not be `2*num_outputs` ?

Reply:
the model predicts only the means, the TestAddStateIndependentNormalScale module adds the scale with the same shape. So instead of 2*num_outputs, the linear layer predicts just num_outputs